.github/workflows/e2e_lighthouse_performance_acceptance_tests.yml

```diff
@@ -282,3 +282,78 @@ jobs:
         with:
           message: "A Lighthouse test failed on the upstream develop branch."
           webhook-url: ${{ secrets.BUILD_FAILURE_ROOM_WEBHOOK_URL }}
+  acceptance_test:
```

Comment:
Can you incorporate the automatic re-build and re-install logic from https://github.com/oppia/oppia/pull/17891?

Reply:
done.